src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -7,6 +7,7 @@
     <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <Nullable>enable</Nullable>
+    <UseAppHost>true</UseAppHost>
```

Comment:
do you want to always add it?
what are the consequences?
(I assume it was committed by mistake)

Reply:
yeah, forgot to unstage it